src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPRequestInvoker.cs

```diff
@@ -129,7 +129,14 @@ JToken ParameterFactory(ITextSnapshot _)
             var responseBody = default(TOut);
             if (response.Response is not null)
             {
-                responseBody = response.Response.ToObject<TOut>(_serializer);
+                if (typeof(TOut) == typeof(JToken))
```

Comment:
Requesting a `JToken` return type is essentially the way to say "I'm gonna handle it myself"